picard/coverart/providers/caa.py

```diff
@@ -94,34 +98,195 @@ def caa_url_fallback_list(desired_size, thumbnails):
     return urls
 
 
+class ArrowButton(QtWidgets.QPushButton):
+    """Standard arrow button for CAA image type selection dialog.
+
+    Keyword Arguments:
+        label {string} -- Label to display on the button
+        command {command} -- Command to execute when the button is clicked (default: {None})
+        parent {[type]} -- Parent of the QPushButton object being created (default: {None})
+    """
+
+    ARROW_BUTTON_WIDTH = 35
+    ARROW_BUTTON_HEIGHT = 20
+
+    def __init__(self, label, command=None, parent=None):
+        if label is None:
```

Comment:
Imho you can make `label` mandatory, and remove those 2 lines.

Reply:
As written, those lines didn't make sense because `label` was already mandatory.  I've removed them.